qiskit/circuit/controlledgate.py

```diff
@@ -84,20 +89,13 @@ def __init__(self, name: str, num_qubits: int, params: List,
            qc2.append(custom, [0, 3, 1, 2])
            qc2.draw()
         """
+        self.base_gate = base_gate.copy()
```

Comment:
If `base_gate` is `None` copying it will fail I think, ("AttributeError: 'NoneType' object has no attribute 'copy'
")
```suggestion
        self.base_gate = None if base_gate is None else base_gate.copy()
```